core/src/main/java/org/elasticsearch/action/admin/indices/shrink/ShrinkRequest.java

```diff
@@ -40,14 +39,11 @@
  */
 public class ShrinkRequest extends AcknowledgedRequest<ShrinkRequest> implements IndicesRequest {
 
-    public static final ObjectParser<ShrinkRequest, ParseFieldMatcherSupplier> PARSER =
-        new ObjectParser<>("shrink_request", null);
+    public static final ObjectParser<ShrinkRequest, Void> PARSER = new ObjectParser<>("shrink_request", null);
     static {
-        PARSER.declareField((parser, request, parseFieldMatcherSupplier) ->
-                request.getShrinkIndexRequest().settings(parser.map()),
+        PARSER.declareField((parser, request, parseFieldMatcherSupplier) -> request.getShrinkIndexRequest().settings(parser.map()),
```

Comment:
s/parseFieldMatcherSupplier/context/ ? Or is there a version of this method without context?